doc/neps/ufunc-overrides.rst

```diff
@@ -482,12 +506,12 @@ are not compatible, i.e., implementations should be something like::
 
         # Option 1: call ufunc directly
```

Comment:
BTW, feel free to declare this out-of-scope for this PR, but maybe we should drop the discussion of "option 2" from the NEP now that we've pretty much settled on "option 1" as recommended?